sig-windows/annual-report-2022.md

```diff
@@ -24,86 +25,87 @@
     - [1981 - Windows Privileged Container Support](https://github.com/kubernetes/enhancements/tree/master/keps/sig-windows/1981-windows-privileged-container-support) - v1.26
     - [2802 - Identify Pod's OS during API Server admission](https://github.com/kubernetes/enhancements/tree/master/keps/sig-windows/2802-identify-windows-pods-apiserver-admission) - v1.25
 
```

Comment:
Does Node log viewer work count?

Also, we did start working on InPlace Pod Vertical Scaling support for Windows, but it did not merge yet (https://github.com/kubernetes/kubernetes/pull/112599)

Reply:
We also had the Windows pod sandbox stats information KEP: https://github.com/kubernetes/enhancements/pull/3439

Reply:
node log viewer was 2023 :D
I added a note about the 2371 work